tests/release_tests/test_launch/test_kubernetes.py

```diff
@@ -0,0 +1,187 @@
+                                                "name": "wandb-api-key",
+                                                "key": "password",
+                                            }
+                                        },
+                                    },
+                                    {
+                                        "name": "AWS_ACCESS_KEY_ID",
+                                        "valueFrom": {
+                                            "secretKeyRef": {
+                                                "name": "aws-access-key-id",
+                                                "key": "password",
+                                            }
+                                        },
+                                    },
+                                    {
+                                        "name": "AWS_SECRET_ACCESS_KEY",
+                                        "valueFrom": {
+                                            "secretKeyRef": {
+                                                "name": "aws-secret-access-key",
+                                                "key": "password",
+                                            }
+                                        },
+                                    },
+                                    {
+                                        "name": "AWS_SESSION_TOKEN",
+                                        "valueFrom": {
+                                            "secretKeyRef": {
+                                                "name": "aws-session-token",
+                                                "key": "password",
+                                            }
+                                        },
+                                    },
+                                ]
+                            }
+                        ]
+                    }
+                }
+            },
+        }
+    },
+}
+
+
+def test_kubernetes_agent_on_local_process():
+    entity = "launch-release-testing"
+    project = "release-testing"
+    queue = "kubernetes-queue"
+    job_name = "job-source-release-testing-train.py:v1"
+
+    try:
+        # Start launch agent
+        agent_process = run_cmd_async(f"wandb launch-agent -q {queue} -e {entity}")
+
+        # Start run
+        queued_run = launch_add(
+            job=f"{entity}/{project}/{job_name}", queue_name=queue, entity=entity
+        )
+
+        # Assert successful
+        run = queued_run.wait_until_finished()
+        assert run.state == "finished"
+    finally:
+        agent_process.kill()
+        run_cmd("rm -r artifacts")
+
+
+@pytest.mark.timeout(180)
+def test_kubernetes_agent_in_cluster(api_key, base_url):
+    entity = "launch-release-testing"
+    project = "release-testing"
+    queue = "kubernetes-queue"
+    job_name = (
+        "job-305054156030.dkr.ecr.us-east-2.amazonaws.com_release-testing_latest:v0"
```

Comment:
what is the difference between this job and the job in the test before? Maybe a comment? or if they do similar things maybe we only have one job, specified at the top? Or a `mark.parametrize` to loop over multiple kinds of jobs?

Reply:
And maybe a comment describing with the job does?

Reply:
The jobs do different things, but there's no reason they need to. I'll update the first test to use the same job as this one. I'll also give the job a more descriptive name - it's the [fashion mnist job](https://github.com/wandb/launch-jobs/tree/main/jobs/fashion_mnist_train) from our examples.